app/src/main/res/values/strings.xml

```diff
@@ -136,7 +136,7 @@
     <string name="settings_hide_app_title">Hide the Magisk app</string>
     <string name="settings_hide_app_summary">Install a proxy app with random package ID and custom app label</string>
     <string name="settings_restore_app_title">Restore the Magisk app</string>
-    <string name="settings_restore_app_summary">Un-hide the app and restore it back to the original APK</string>
```

Comment:
This unreferences one from the other, so that it has to be inferred.

Reply:
I disagree; it's technically correct.

Reply:
Undisputed. My problem is that it is less clear and demands more of the user to understand it.

Reply:
Actually it's more accurate for the user by explicitly saying the original APK was changed.

Reply:
Great point. Saying "restore" twice is not productive.

Reply:
OK